frontend/javascripts/admin/organization/organization_edit_view.js

```diff
@@ -0,0 +1,175 @@
+// @flow
+import { withRouter } from "react-router-dom";
+import { Form, Button, Card, Input, Icon, Row } from "antd";
+import React from "react";
+
+import { confirmAsync } from "dashboard/dataset/helper_components";
+import { getOrganization, deleteOrganization, updateOrganization } from "admin/admin_rest_api";
+import Enum from "Enumjs";
+
+const FormItem = Form.Item;
+
+export const PricingPlanEnum = Enum.make({
+  Basic: "Basic",
+  Premium: "Premium",
+  Pilot: "Pilot",
+  Custom: "Custom",
+});
+type PricingPlan = $Keys<typeof PricingPlanEnum>;
+
+type Props = { organizationName: string, form: Object };
+
+type State = {
+  displayName: string,
+  newUserMailingList: string,
+  pricingPlan: ?PricingPlan,
+  isFetchingData: boolean,
+  isDeleting: boolean,
+};
+
+class OrganizationEditView extends React.PureComponent<Props, State> {
+  state = {
+    displayName: "",
+    newUserMailingList: "",
+    pricingPlan: null,
+    isFetchingData: false,
+    isDeleting: false,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  async fetchData() {
+    if (this.props.organizationName) {
+      this.setState({ isFetchingData: true });
+      const { displayName, newUserMailingList, pricingPlan } = await getOrganization(
+        this.props.organizationName,
+      );
+      this.setState({
+        displayName,
+        pricingPlan: Enum.coalesce(PricingPlanEnum, pricingPlan),
+        newUserMailingList,
+        isFetchingData: false,
+      });
+    }
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    this.props.form.validateFields(async (err, formValues) => {
+      if (!err) {
+        await updateOrganization(
+          this.props.organizationName,
+          formValues.displayName,
+          formValues.newUserMailingList,
+        );
+        window.location.replace(`${window.location.origin}/dashboard/datasets`);
```

Comment:
```suggestion
        window.location.replace(`${window.location.origin}/dashboard/`);
```

That way, it should go to the least recently used tab.